mcc/got/cmd/delete.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// deleteCmd represents the delete command
+var deleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
```

Comment:
Yes, it's automatically generated. I didn't change it yet because the subcommand has been created but it's not implemented. When I implement its functionality I remove the default message and write proper help.